nodejs/eks/workerpool.ts

```diff
@@ -114,32 +124,77 @@ export class WorkerPool extends pulumi.ComponentResource {
     constructor(name: string, args: WorkerPoolOptions, opts?: pulumi.ComponentResourceOptions) {
         super("eks:index:WorkerPool", name, args, opts);
 
-        const pool = createWorkerPool(name, args, this, this.getProvider("kubernetes:core:v1/ConfigMap"));
+        const pool = createWorkerPool(name, args, this);
         this.instanceRole = pool.instanceRole;
         this.nodeSecurityGroup = pool.nodeSecurityGroup;
-
-        this.registerOutputs();
```

Comment:
This call is required in order to inform the CLI's display routine that the component has finished creating. Let's keep it as-is.